subprojects/core/src/main/java/org/gradle/process/internal/DefaultExecHandleBuilder.java

```diff
@@ -31,46 +35,39 @@
  */
 @SuppressWarnings("DeprecatedIsStillUsed")
 @Deprecated
-public class DefaultExecHandleBuilder extends AbstractExecHandleBuilder implements ExecHandleBuilder, ProcessArgumentsSpec.HasExecutable {
+public class DefaultExecHandleBuilder extends AbstractExecHandleBuilder implements ExecHandleBuilder {
```

Comment:
Sounds like something to add as a comment in the code. I see this is already `@Deprecated`, so I guess it will be gone in 9.0 anyway.

Can we at least remove the `@SuppressWarnings("DeprecatedIsStillUsed")`? Based on your comment I would expect the class _not_ to be used in Gradle itself, right?